sklearn/feature_extraction/text.py

```diff
@@ -133,12 +133,22 @@ def _word_ngrams(self, tokens, stop_words=None):
         min_n, max_n = self.ngram_range
         if max_n != 1:
             original_tokens = tokens
-            tokens = []
+            if min_n == 1:
+                tokens = list(original_tokens)
+                min_n += 1
```

Comment:
Maybe just add a comment here to say that this does the same thing as the fist iteration of the loop below (which is then skipped) as that's not entirely clear from reading it.